src/components/App.js

```diff
@@ -41,9 +43,19 @@ const App = ({...props}) => {
   const languageSwitcher = translationEnabled ? <LanguageSwitcher /> : null;
   const router = (
     <Routes>
-      {/* Anything dealing with appointments gets routed to it's own sub-router */}
+      {/* Anything dealing with appointments gets routed to its own sub-router */}
       <Route path="afspraak-annuleren/*" element={<ManageAppointment />} />
       <Route path="appointment/*" element={<AppointmentForm {...props} />} />
+
+      <Route
+        path="/cosign/check"
+        element={
+          <ErrorBoundary useCard>
+            <CosignSummary {...props} />
+          </ErrorBoundary>
+        }
+      />
```

Comment:
This will change in the next PR because there will be one more view. So I think i'll move this to another component (like the `AppointmentForm` does above).